app/templates/search.html

```diff
@@ -1,14 +1,39 @@
 {% extends "base.html" %}
 {% import '_snippets.html' as snippets %}
+{% block scripts %}
+  <script>
```

Comment:
You told me that. If we go to search page that navbar search bar should go off and only search box in page body should appear. 
The nav bar is in base.html and every other page extends that. So the search page has the same nav bar with search box. We wanted to remove navbar search box go disappear like github page.